app/controllers/state_file/questions/nj_homeowner_eligibility_controller.rb

```diff
@@ -6,20 +6,25 @@ class NjHomeownerEligibilityController < QuestionsController
       before_action -> { @filing_year = Rails.configuration.statefile_current_tax_year }
 
       def self.show?(intake)
-        intake.household_rent_own_own?
+        intake.household_rent_own_own? &&
+        Efile::Nj::NjPropertyTaxEligibility.determine_eligibility(intake) != Efile::Nj::NjPropertyTaxEligibility::INELIGIBLE
       end
 
       def next_path
         options = {}
         options[:return_to_review] = params[:return_to_review] if params[:return_to_review].present?
-
+        
         case StateFile::NjHomeownerEligibilityHelper.determine_eligibility(current_intake)
         when StateFile::NjHomeownerEligibilityHelper::INELIGIBLE
           NjIneligiblePropertyTaxController.to_path_helper(options)
         when StateFile::NjHomeownerEligibilityHelper::UNSUPPORTED
           NjUnsupportedPropertyTaxController.to_path_helper(options)
         else
-          NjHomeownerPropertyTaxController.to_path_helper(options)
+          if Efile::Nj::NjPropertyTaxEligibility.determine_eligibility(current_intake) == Efile::Nj::NjPropertyTaxEligibility::INELIGIBLE_FOR_DEDUCTION
+            super
```

Comment:
[DUST] If adding a comment, this is the confusing part. When the applicant is INELIGIBLE_FOR_DEDUCTION, it calls next_path()? If adding comment, perhaps clarify how this is meant to skip certain eligibility questions.

Reply:
This PR add three uses of `super` within `next_path`, so I just added comments to those three lines to make more clear what the behavior is